Edited assets API files to be consistent with API documentation stand… (#561)
osisoft/OCS-Docs: content/api-reference/assets/asset-status.md

| Property            | Type                     | Required? | Searchable? | Description                                                  |
| ------------------- | ------------------------ | --------- | ----------- | ------------------------------------------------------------ |

> | StreamReferenceId   | String                   | Required  | No          | `Id` for the asset's StreamReference property. <!-- Anthony: Should this be StreamReferences? -->The stream reference must exist before the status mapping can be created. |

Reviewer: Yeah, it makes more sense to make this StreamReferences.

Author: done

> | StreamReferenceId   | String                   | Required  | No          | `Id` for the asset's StreamReference property. <!-- Anthony: Should this be StreamReferences? -->The stream reference must exist before the status mapping can be created. |

Author: ```suggestion
| StreamReferenceId   | String                   | Required  | No          | `Id` for the asset's StreamReferences property. The stream reference must exist before the status mapping can be created. |
```

## `Get Asset Status`

Returns the status of an asset. 

The status of an asset is determined by an exact match of the SDS stream property value to the value of the `ValueStatusMappings`. If there are no exact matches, the status is a 0 (Unknown).

> <!-- Anthony: Would it be correct and clearer to change "SDS stream property value" above to "SDS `StreamPropertyID` value"? -->

Reviewer: What the above is saying is that we want to update the status with respect to the current value of the Sds Stream Property. This shouldn't be the property id.

> <!-- Anthony: Would it be correct and clearer to change "SDS stream property value" above to "SDS `StreamPropertyID` value"? -->

Author: ```suggestion
```

| Status Code      | Body Type         | Description                                                  |
| ---------------- | ----------------- | ------------------------------------------------------------ |

> | 200 OK           | Asset status list | On successful `POST`, returns the status of multiple assets. |

Reviewer 2: Anywhere you call out the HTTP request method (`POST`, `GET`), I think you should add request afterward.

Author: @atangOSIsoft Hi Anthony, Can you please reply?

Reviewer 2: No, I meant  to add 'request' in the description: On successful POST request

Reviewer: OH! I see, you mean the text, not actually the content. Yeah, I believe it reads better as "On successful `POST` request, returns the status of multiple assets." We can also just assume that the 200 means all good and put the description as "Returns the status of multiple assets." For 207, we can put : "Partial success. Not all assets were able to be resolved. See response body for additional details."

> | 200 OK           | Asset status list | On successful `POST`, returns the status of multiple assets. |

Author: ```suggestion
| 200 OK           | Asset status list | Returns the status of multiple assets. |
```

> | 207 Multi Status | Multi Status      | On `POST`, returns the status of multiple assets. For error responses, check the multi-status response for the error and cause. |

Author: ```suggestion
| 207 Multi Status | Multi Status      | Partial success. Not all assets were able to be resolved. See response body for additional details. |
```
